uwsdora.module

```diff
@@ -106,3 +106,54 @@ function uwsdora_views_api() {
     'path' => drupal_get_path('module', 'uwsdora') . '/views',
   );
 }
+
+/**
+ * Implements hook_convert_mods_to_citeproc_jsons_alter().
+ *
+ * @see citeproc/includes/converter.inc::convert_mods_to_citeproc_json_page()
+ */
+function uwsdora_convert_mods_to_citeproc_jsons_alter(&$output, $mods) {
+  $data = function($mods) {
```

Comment:
`$data` isn't... a great variable name, especially for a function. `$extract_page_info` or something of the like, maybe?